### ISSUE TITLE
Standardize nodata value output format

### ISSUE THREAD

**Issue by virginiayung** [mapbox/rio-alpha#15]
### Tl;dr

ndv values need to follow the standard format: `"[int(ndv), int(ndv), int(ndv)]"` or `int(ndv)`. 
# 

Currently, there are two ways to represent ndv in pxm and we rely solely on bash to handle the parsing of ndv strings:

1) single integer
2) 3 separate integers (For example: The ndv values for [chilliwack_ortho](https://github.com/mapbox/pxm-sources/blob/4dd0519c06a3fab559c5aa85778f2885755ca16e/chilliwack_ortho/ndv#L1) are `255 255 255`)
The second case is throwing a parsing error when we integerate rio-alpha into [pxm-prep](https://github.com/mapbox/pxm/blob/b58845b3b81fa67b18f3d57749b8dced06978c44/pxm-prep#L183), causing [`pxm-prep.test.sh` to fail](https://github.com/mapbox/pxm/pull/627):

<img width="620" alt="screen shot 2016-08-29 at 1 08 45 pm" src="https://cloud.githubusercontent.com/assets/4450007/18068783/fd58eca6-6df7-11e6-95e3-047cd0447b65.png">
## Why is this happening?

This is caused by two different ndv values formats:
#### rio islossy accepts `"[255, 255, 255]"`, while rio findnodata outputs `"255 255 255"`

`islossy` takes either 1) a single integer or 2) a string of a list of 3 integers that are separated by `,`. It uses two functions  [_parse_ndv and `_parse_single`](https://github.com/mapbox/rio-alpha/blob/master/rio_alpha/utils.py#L9-L51) to handle both cases and parse the nodata values. We did not rely on Click to handle both input cases (single integer vs. string of list of 3 integers) because Click option only supports a fixed number of arguments. If we set `--ndv` to have `nargs=3`, it will throw an error if I pass it only a single integer and vice versa:

<img width="928" alt="screen shot 2016-08-29 at 3 25 35 pm" src="https://cloud.githubusercontent.com/assets/4450007/18069573/e7085036-6dfc-11e6-9438-5ef8c47a3d99.png">
## Fix

We have a couple options:
1) Change [pxm-prep script](https://github.com/mapbox/pxm/blob/master/pxm-prep#L151-L181) and `rio findnodata` output format from `"255 255 255"` to `"[255, 255, 255]"`.
    Similar to [pxm-islossy.test.sh](https://github.com/mapbox/pxm/blob/b58845b3b81fa67b18f3d57749b8dced06978c44/test/pxm-islossy.test.sh#L15-L18):

```
from `ndv="$1 $2 $3"` and  `--ndv $ndv`

to `ndv="[$1, $2, $3]"` and `--ndv "$ndv"`
```

2) Change [pxm-prep script](https://github.com/mapbox/pxm/blob/master/pxm-prep#L151-L181) and  `rio findnodata` output format from `255 255 255` => `"255 255 255"`, also change `rio islossy` input format from `"[255, 255, 255]"` to `"255 255 255"`

```
from `--ndv $ndv` 
to `--ndv "$ndv"`
```

3) ~~We can then pass any number of ndv by utilizing click multiple input options:~~
  ~~Change rio islossy input from `--ndv 255  --ndv 255  --ndv 255`~~
 ~~and pxm to  `--ndv $1 --ndv $2 --ndv $3`. ~~
This method is too messy

cc @jacquestardie
